torchvision/prototype/transforms/functional/_augment.py

```diff
@@ -1,52 +1,68 @@
 from typing import Tuple
+from typing import TypeVar
 
 import torch
-from torchvision.transforms import functional as _F
+from torchvision.prototype import features
+from torchvision.prototype.transforms import kernels as K
 
+from ._utils import dispatch
 
-erase_image = _F.erase
+T = TypeVar("T", bound=features.Feature)
 
 
-def _mixup(input: torch.Tensor, batch_dim: int, lam: float, inplace: bool) -> torch.Tensor:
-    if not inplace:
-        input = input.clone()
+@dispatch(
+    {
+        features.Image: K.erase_image,
+    },
+)
```

Comment:
Matter of taste question, can we remove trailing commas to make this inline ? 4 lines to create a dispatcher for images looks a bit ugly.
Otherwise, we can precode the functions inside dispatch constructor: (op_image, op_bbox, op_segm_mask, etc).

Reply:
I agree for dispatchers with a single kernel, but I would like to keep the format for more kernels.